pkg/rpcclient/rpc_test.go

```diff
@@ -2022,6 +2030,7 @@ func TestUninitedClient(t *testing.T) {
 
 	c, err := New(context.TODO(), endpoint, opts)
 	require.NoError(t, err)
+	close(startSending)
```

Comment:
Also, I don't think we need this fix for non-WS client, because standard RPC client doesn't open connection on client creation. @roman-khimov, agree?

Reply:
Yes.